charts/argo-cd/values.yaml

```diff
@@ -3019,6 +3019,12 @@ notifications:
     # -- Whether helm chart creates notifications controller secret
     create: true
 
+    # -- notifications controller Secret name
+    name: "argocd-notifications-secret"
```

Comment:
I dont think that this is configurable. And when it is, we need to pass it somehow to the notification controller, he is talking to the kube-api to read that secret directly.